.circleci/config.yml

```diff
@@ -5,6 +5,7 @@ orbs:
   shellcheck: circleci/shellcheck@1.2.0
   slack: circleci/slack@0.1.14
   windows: circleci/windows@2.2.0
+  nancy: sonatype-nexus-community/circleci-nancy-orb@0
```

Comment:
The tagging of the orb version makes me a little nervous. Since there is no 1.0 version (is this in the works?), we should absolutely lock this to the patch version.